jdbi/src/main/java/org/killbill/commons/jdbi/argument/LocalDateArgumentFactory.java

```diff
@@ -48,10 +48,9 @@ public LocalDateArgument(final LocalDate value) {
         @Override
         public void apply(final int position, final PreparedStatement statement, final StatementContext ctx) throws SQLException {
             if (value != null) {
-                // ISO8601 format
-                statement.setString(position, value.toString());
+                statement.setDate(position, new java.sql.Date(value.toDate().getTime()));
```

Comment:
String parameters do not implicitly cast: "org.postgresql.util.PSQLException: ERROR: column "invoice_date" is of type date but expression is of type character varying"

I started with a Postgres-specific solution (https://github.com/marksimu/killbill-commons/commit/cfed19d889a21191928dc0fc47e44e8d758b6991) but that seemed to be hacky and unnecessary.  Will look into this further.